dts/xtensa/intel/intel_cavs20.dtsi

```diff
@@ -125,8 +125,9 @@
 			interrupt-parent = <&core_intc>;
 		};
 
-		idc: idc@1200 {
-			compatible = "intel,cavs-idc";
+		intel_adsp_idc: idc@1200 {
+			compatible = "intel,adsp-idc";
+			label = "CAVS_IDC";
```

Comment:
labels are now not required and being cleaned up in the tree, this also keeps the CAVS in the name, which is beling replaced everywhere else.